src/calendar.cpp

```diff
@@ -398,6 +398,14 @@ int calendar::season_length()
     return cached_season_length;
 }
 
+int calendar::turn_of_year() const {
+	return (season_turns() * season) + (turn_number % season_turns());
```

Comment:
uhm, the new function would return DAYS( season_length() + turns_into_current_season which equals turn_number